examples/machine_learning/neural_network_train.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
```

Comment:
I suggest creating "other" directory to hold the sources and "assets" subdirectory inside it where the data would reside

Reply:
Sounds like a good idea. What do you think about editing our post-save hook to drop the .py files in a centralized `other` (maybe we call this src) directory (preserving folder order)? So, the directory structure would end up looking like this:

--job
--machine_learning
--material
--system
--workflow
--assets
--other (or src)
||--job
||--machine_learning
||--material
||--system
||--workflow

Reply:
New directory structure is created in most recent commits